docs/1.0.1/html/about.html

```diff
@@ -1,588 +1 @@
-<li class="toctree-l2"><a class="reference internal" href="#assembler-syntax-descriptions">Assembler syntax descriptions</a></li>
-<li class="toctree-l2"><a class="reference internal" href="#feedback">Feedback</a></li>
-</ul>
-</li>
-</ul>
-<ul>
-<li class="toctree-l1"><a class="reference internal" href="overview/intro.html">1. Introduction</a></li>
-<li class="toctree-l1"><a class="reference internal" href="overview/goals.html">2. Design goals</a></li>
-<li class="toctree-l1"><a class="reference internal" href="overview/functionality.html">3. Functionality overview</a></li>
-<li class="toctree-l1"><a class="reference internal" href="overview/sample-arch.html">4. Sample architectures</a></li>
-<li class="toctree-l1"><a class="reference internal" href="overview/conventions.html">5. Library conventions</a></li>
-<li class="toctree-l1"><a class="reference internal" href="overview/implementation.html">6. Implementation considerations</a></li>
-<li class="toctree-l1"><a class="reference internal" href="overview/usage.html">7. Usage considerations</a></li>
-<li class="toctree-l1"><a class="reference internal" href="api/library/index.html">8. Library management reference</a></li>
-<li class="toctree-l1"><a class="reference internal" href="api/keys/index.html">9. Key management reference</a></li>
-<li class="toctree-l1"><a class="reference internal" href="api/ops/index.html">10. Cryptographic operation reference</a></li>
-</ul>
-<ul>
-<li class="toctree-l1"><a class="reference internal" href="appendix/example_header.html">Example header file</a></li>
-<li class="toctree-l1"><a class="reference internal" href="appendix/specdef_values.html">Example macro implementations</a></li>
-<li class="toctree-l1"><a class="reference internal" href="appendix/history.html">Changes to the API</a></li>
-</ul>
-<ul>
-<li class="toctree-l1"><a class="reference internal" href="psa_c-identifiers.html">Index of API elements</a></li>
-</ul>
-<div id="searchbox" style="display: none" role="search">
-  <h3 id="searchlabel">Quick search</h3>
-    <div class="searchformwrapper">
-    <form class="search" action="search.html" method="get">
-      <input type="text" name="q" aria-labelledby="searchlabel" />
-      <input type="submit" value="Go" />
-    </form>
-    </div>
-</div>
-<script type="text/javascript">$('#searchbox').show(0);</script>
-        </div>
-      </div>
-      <div class="clearer"></div>
-    </div>
-    <div class="footer">
-      &copy; 2018-2020, Arm Limited or its affiliates. All rights reserved.
-      
-      |
-      Powered by <a href="http://sphinx-doc.org/">Sphinx 2.1.2</a>
-      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.12</a>
-      
-    </div>
-
-    
-
-    
-  </body>
-</html>
\ No newline at end of file
+<meta http-equiv="Refresh" content="0; url='https://arm-software.github.io/psa-api/crypto/1.0/about.html'" />
```

Comment:
I'd prefer if we included some courtesy information for non-interactive clients, eg:
```suggestion
<html>
  <head>
    <title>Redirecting to https://arm-software.github.io/psa-api/crypto/1.0/api/library/index.html</title>      
    <meta http-equiv="Refresh" content="0; url='https://arm-software.github.io/psa-api/crypto/1.0/about.html'" />
  </head>
  <body>
    <p>This page has moved to <a href="https://arm-software.github.io/psa-api/crypto/1.0/api/library/index.html">
      https://arm-software.github.io/psa-api/crypto/1.0/api/library/index.html</a>.</p>
  </body>
</html>
```